scripts/processFactory.js

```diff
@@ -0,0 +1,1911 @@
+var ts;
```

Comment:
I don't think we actually check in the js files into scripts, and if we do, we probably shouldn't.